.github/workflows/publish-docs-preview.yml

```diff
@@ -0,0 +1,58 @@
+name: Publish docs preview
+on:
+  pull_request_target:
+    types: labeled
+
+jobs:
+
+  publish-docs-preview:
+    name: Publish docs preview
+    runs-on: ubuntu-22.04
+    if: ${{ github.event.label.name ==  ':eyes: publish-docs-preview' }}

```

Comment:
Not sure if I must use the shortcode or the real emoji character here. I entered the name with the shortcode, but it might be converted... we'll see.